python/cudf/cudf/core/column/column.py

```diff
@@ -960,13 +960,9 @@ def view(self, dtype: Dtype) -> ColumnBase:
 
         Parameters
         ----------
-        dtype : NumPy dtype, string
+        dtype : NumPy dtype
```

Comment:
This can also be one of our extension types, right (hence why we're typing with DtypeObj)?

Reply:
Yup good point. Updated the docstring here